pallets/parachain-staking/src/benchmarks.rs

```diff
@@ -35,12 +66,112 @@ benchmarks! {
 	}: _(RawOrigin::Root, inflation_range)
 	verify {
 	}
+
+	join_candidates {
+		let caller: T::AccountId = create_funded_user::<T>("caller", USER_SEED, 100);
+		whitelist_account!(caller); // TODO: why is this line necessary, copy pasta-ed
```

Comment:
I don't recall where I saw this documented, but IIRC it it's related to telling the benchmarking pipeline that a lookup associated with this account shouldn't impact the weight results. That's not a high confidence answer, we should hunt down the documentation :)